src/web/layouts/LiveLayout.tsx

```diff
@@ -480,6 +377,9 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 							/>
 						</div>
 					</GridItem>
+					<GridItem area="border">
+						<></>
+					</GridItem>
```

Comment:
The same as Immersives, we don't show the grey border but we do still want the vertical spacing so we use this pattern